src/modules/flow/iio/nodes.c

```diff
@@ -959,6 +959,160 @@ light_open(struct sol_flow_node *node, void *data, const struct sol_flow_node_op
 }
 
 static bool
+intensity_both_create_channels(struct iio_double_data *mdata, int device_id)
+{
+    mdata->iio_base.device = sol_iio_open(device_id, &mdata->iio_base.config);
+    SOL_NULL_CHECK(mdata->iio_base.device, false);
+
+    mdata->channel_val = iio_add_channel(mdata->scale, mdata->offset, "in_intensity0_both", &mdata->iio_base);
```

Comment:
@edersondisouza The current GEN_CHANNEL_NAME macro is used to append a "%d" at the end of the string. But the two in_intensity0_both and in_intensity0_ir have a different format(%d is in hte middle of the string), any thoughts/suggestions about the change?